waves/project.py

```diff
@@ -1592,19 +1594,25 @@ def energy_losses(
             by="turbine",
             units="kw",
         )
+
         production = self.energy_production(  # type: ignore
```

Comment:
I'm realizing now that `energy_production` having losses applied would create a cyclical calculation, so we'll need to reconfigure this calculation just a little bit. So we'd need to start from the the energy potential, then calculate the loss ratios for each category, then apply them to the losses according the ORCA calculation.

Essentially what's happening in the energy production calculation is the application of the wake and availability losses, so we just have to pull them out independentially instead of calling `energy_production`.

Reply:
Now, we can just get the energy potential and subtract the combined wake losses, availability, technical losses, and environmental losses (total losses)

Reply:
Adressed by eliminating energy_production and  doing energy_potential * total_loss_ratio